src/ducks/contents.js

```diff
@@ -26,13 +26,16 @@ const projectContentsReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchProjectContents = (project_id) => {
+const fetchProjectContents = (id, type) => {
+  type = type ? type.split('/')[0] : 'project';
   return (dispatch) => {
     dispatch({
       type: FETCH_PROJECT_CONTENTS,
```

Comment:
Good point.

I was thinking that this should be a generic loader for any of the resources listed in #6, so that new resource types can be added without having to write a lot of new code.

Maybe we should define a `TranslationResource` component (I'm not sure about the name), which is a generic adapter for specific translations, something like [the PFE file viewer](/zooniverse/Panoptes-Front-End/blob/master/app/components/file-viewer/index.jsx).